packages/shared/refThrottled/index.ts

```diff
@@ -10,18 +14,41 @@ import { useThrottleFn } from '../useThrottleFn'
  * @param  delay  A zero-or-greater delay in milliseconds. For event callbacks, values around 100 or 250 (or even higher) are most useful.
  * @param [trailing] if true, update the value again after the delay time is up
  * @param [leading] if true, update the value on the leading edge of the ms timeout
+ * @param [deep] default true, Depth monitoring, if false, only monitor the first level
+ * @param [immediate] default true, Whether to execute immediately
+ * @param [cloneHandler] By default, it use `JSON.parse(JSON.stringify(value))` to clone
  */
-export function refThrottled<T>(value: Ref<T>, delay = 200, trailing = true, leading = true) {
+export function refThrottled<T>(value: Ref<T>, delay = 200, trailing = true, leading = true, deep = true, immediate = true, cloneHandler = cloneFnJSON) {
```

Comment:
I think that this should take an options object here instead of adding more parameters.

Reply:
I'm trying.